x-pack/plugins/alerting/server/backfill_client/backfill_client.ts

```diff
@@ -56,12 +82,12 @@ export class BackfillClient {
     unsecuredSavedObjectsClient,
   }: BulkQueueOpts): Promise<ScheduleBackfillResults> {
     const adHocSOsToCreate: Array<SavedObjectsBulkCreateObject<AdHocRunSO>> = [];
-    const resultOrErrorMap: Map<number, number | ScheduleBackfillError> = new Map();
+    const soToCreateOrErrorMap: Map<number, number | ScheduleBackfillError> = new Map();
```

Comment:
Yea, I agree it is a little confusing. I want to return a response for each schedule request in the same order that I receive them and there are so many error cases to account for 😅. I can try to make it clearer.

Reply:
comment is good for now, thx!

I'm not sure about failing the entire request for individual rule issues.  Doesn't feel right to me, but it may be that we have a pattern like that already?  I would think, as a user, I'd want anything that could be scheduled to be scheduled, and then errors reported for things that couldn't be.